wiki/Mascots/Gallery/es.md

```diff
@@ -28,47 +24,47 @@ needs_cleanup: true
 
 ![pippi rediseñada por Daru](img/pippi-Daru-redesigned.png "pippi rediseñada por Daru (vista en la página de bienvenida)")
 
-![comboburst de osu! de la skin predeterminada v1](img/pippi-Daru-comboburst.png "Comboburst de la skin v2 predeterminada.")
+![Combo burst de osu! de la skin predeterminada v2](img/pippi-Daru-comboburst.png "Combo burst de la skin predeterminada v2.")
 
 ![Imagen de error 404](img/pippi-Daru-404.png "Imagen de error 404")
 
-#### Arte de fans
+#### Fan arts
 
-Para arte de fans de pippi, véase:
+Para los fan arts de pippi, véase:
 
-- [First Official osu! Fanart Contest Results! (Facebook)](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152186520298282)
-- [osu! Fanart Contest 2! (Facebook)](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152291754173282)
-- [osu! Fanart Contest 3! (Facebook)](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152514671088282)
+- [First Official osu! Fanart Contest Results! (Facebook)](https://www.facebook.com/media/set/?set=a.10152186520298282)
+- [osu! Fanart Contest 2! (Facebook)](https://www.facebook.com/media/set/?set=a.10152291754173282)
+- [osu! Fanart Contest 3! (Facebook)](https://www.facebook.com/media/set/?set=a.10152514671088282)
```

Comment:
haz un pr del inglés / demás

Reply:
¿Segura? Solo es por los links, sería mejor si fuese para limpiar el artículo

Reply:
(i might be misunderstanding cuz machine translation)  if fixing the links here would also be considered a fix for any other language, then yes it should be applied to EN and other translations

Reply:
> (i might be misunderstanding cuz machine translation) if fixing the links here would also be considered a fix for any other language, then yes it should be applied to EN and other translations

Well, I didn't change them in the other languages because the English article is marked as `Needs cleanup` and I thought that if a change will be made, it would be to clean up the article. But sure, let me change the links in the other languages